Signal/translations/bin/auto-genstrings

```diff
@@ -4,15 +4,6 @@ set -e
 
 SSK_DIR="../SignalServiceKit/src"
 
-pushd $SSK_DIR
```

Comment:
I get that this is annoying as is. But how are you going to ensure that you don't push the wrong strings to transifex?